<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WQTimerCtrl</class>
 <widget class="QWidget" name="WQTimerCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>11</number>
   </property>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>快退</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QDateTimeEdit" name="dateTimeEdit_2">
     <property name="displayFormat">
      <string>yyyy-MM-dd HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>结束时间：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="displayFormat">
      <string>yyyy-MM-dd HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>快进</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>开始时间：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>设置偏移</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
